### PR TITLE
ref(sampling): Send uniform rule always last position array - [TET-164]

### PR DESCRIPTION
**Pull request by priscilawebdev**
What this PR does:

- Update the submit logic of the "Specific conditions" Modal to always send the uniform rule in the last position of the array
- Create tests for it 
- Add an "aria-label-disabled" to the draggable item
- Update indicator error message 

A bit more

- Moved all the draggable list items to the server-side-sampling folder (I can open a separate PR if needed) - No code changes

**Preview:**

image

### CODE REVIEW

File: static/app/views/settings/project/server-side-sampling/modals/specificConditionsModal/index.tsx [hunk 1]
```diff
@@ -153,12 +154,18 @@ export function SpecificConditionsModal({
       ? rules.map(existingRule => (existingRule.id === rule.id ? newRule : existingRule))
       : [...rules, newRule];
 
+    // Make sure that a uniform rule is always send in the last position of the rules array
+    const [uniformRule, specificRules] = partition(newRules, isUniformRule);
```
Comment (jjbayer) [inline static/app/views/settings/project/server-side-sampling/modals/specificConditionsModal/index.tsx:158]: I'm not sure if partitioning conditions into "uniform" and "specific" will fix all the cases. Imagine rules

* `release=foo -> 20%`
* `release=foo environment=bar -> 100%`
* `all -> 1%`

`all -> 1%` would be moved to the bottom by this logic, but `release=foo -> 20%` would still precede the more specific `release=foo environment=bar -> 100%`.

Automatically determining which rule should have precedence is hard (which is why we don't even attempt to do so in Relay). So to be honest I would just give the user a rearrangeable list of rules and make it clear to them that the first matching rule wins.

Comment (priscilawebdev) [reply static/app/views/settings/project/server-side-sampling/modals/specificConditionsModal/index.tsx:158]: Yes, but we are doing it. In the UI, the uniform rules always need to be fixed at the bottom and cannot be dragged (else operator), while the other rules can be dragged and the UI shows for each one an operator "if and else if". The idea behind the operator was to display the order of priority

Comment (priscilawebdev) [reply static/app/views/settings/project/server-side-sampling/modals/specificConditionsModal/index.tsx:158]: **if** release=foo -> 20%
**else if** release=foo environment=bar -> 100%
**else** all -> 1

Comment (jjbayer) [reply static/app/views/settings/project/server-side-sampling/modals/specificConditionsModal/index.tsx:158]: OK, so it's a special case reordering for `else`. That makes sense I guess.

File: static/app/stores/serverSideSamplingStore.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import {createStore} from 'reflux';
```
Comment (matejminar) [inline static/app/stores/serverSideSamplingStore.tsx:1]: Is this new file here on purpose or is it just a leftover from a different feature?

Comment (priscilawebdev) [reply static/app/stores/serverSideSamplingStore.tsx:1]: I pushed it when solving conflict. good catch

**Review by matejminar — APPROVED** [review]

**Pull request merged**